jenkins/scripts/integration_test_env.sh

```diff
@@ -16,6 +16,7 @@ then
     export BAREMETAL_OPERATOR_LOCAL_IMAGE="https://github.com/metal3-io/baremetal-operator.git"
     export CAPM3_LOCAL_IMAGE="https://github.com/metal3-io/cluster-api-provider-metal3.git"
     export CAPM3_LOCAL_IMAGE_BRANCH="${CAPM3RELEASEBRANCH}"
+    export BMOBRANCH="main"
```

Comment:
can you please clarify the comment (main test to periodic) for this condition check on line 12?

Reply:
I think this part is responsible for the main test only and need to change the title which should refer to main test only.

Reply:
@smoshiur1237 I didnt get you, the title already says that

Reply:
@furkatgofurov7 done

Reply:
I got confuse with the main and periodic test. But all good from my side.